scripts/spell-check.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#
+## Copyright (C) 2020 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+#
+# This script runs codespell against selected files.
+#
+# Use -q or --quiet option to run quietly
+#
+
+CODESPELL_LOC=`which codespell`
+if test "${CODESPELL_LOC}" = ""; then
+    echo "This script requires codespell which was not found."
+    exit 1
+fi
+
+CODESPELL_VER=`codespell --version 2>&1`
+echo "The codespell version is ${CODESPELL_VER}."
```

Comment:
I doubt we should add the `-q` option (too much work for too little gain), but if you do add that option, then this output ought to be covered by it IMO.

Reply:
Removed the -q option in the next commit.

Reply:
Thank you. I polished this further in 5523abb. I consider this change request addressed, but if you insist on keeping the output I removed in that commit, I will restore it if another reviewer backs you up. FWIW, I do not think it is a good idea to be so noisy by default -- that noise often obscures (more) important messages.